_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -926,12 +944,13 @@ terraform {
 
   # Only allow this Terraform version. Note that if you upgrade to a newer version, Terraform won't allow you to use an
   # older version, so when you upgrade, you should upgrade everyone on your team and your CI servers all at once.
-  required_version = "= 0.12.6"
+  required_version = "= 0.12.29"
 }
 ----
 
 Next, use the `account-baseline-root` module from the Gruntwork Infrastructure as Code Library, making sure to replace the `<VERSION>` placeholder
-with the latest version from the https://github.com/gruntwork-io/module-security/releases[releases page]:
+with the latest version from the https://github.com/gruntwork-io/module-security/releases[releases page] (minimum
+version for this guide is `v0.34.1`):
 
 .infrastructure-modules/landingzone/account-baseline-root/main.tf
 [source,hcl]
```

Comment:
NIT: Maybe use this for all the `.tf` files, so we don't have `3` in `s3` highlighted weirdly. Do we have syntax highlighting for terraform? (Ignore if not.)
```suggestion
[source,terraform]
```

Reply:
Agreed the highlighting for `3` is weird.

@eak12913 @oredavids Do we have syntax highlighting for `terraform` or only `hcl` in whatever library we use to do syntax highlighting for AsciiDoc?

FWIW, my editor (IntelliJ) recognized `hcl` with AsciiDoc and highlights those code snippets correctly while I'm editing, but does not recognize `terraform`, so I wonder if only the former actually works?

Reply:
My understanding is that Jekyll uses [Rouge](http://rouge.jneen.net/) as the default highlighter. That highlighter has `terraform` as an language but does not seem to have `hcl`

Reply:
Hm, it seems to be the opposite? When I have `[source,hcl]`, syntax highlighting works. When I switch to `[source,terraform]`, there's no syntax highlighting at all.